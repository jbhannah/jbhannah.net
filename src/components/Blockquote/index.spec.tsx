import { render, screen } from "@testing-library/react"
import * as React from "react"
import { Blockquote } from "."

const TestBlockquote = (props = {}) =>
  render(<Blockquote data-testid="blockquote" {...props} />)

describe("Blockquote", () => {
  beforeEach(() => TestBlockquote())

  it("renders correctly", () => {
    expect(screen.getByTestId("blockquote")).toMatchSnapshot()
  })
})
