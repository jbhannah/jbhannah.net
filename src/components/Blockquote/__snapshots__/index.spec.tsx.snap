// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blockquote renders correctly 1`] = `
<blockquote
  class="css-2570gk"
  data-testid="blockquote"
/>
`;
